wiki/osu!_wiki_Contribution_Guide/Common_Issues/en.md

```diff
@@ -48,6 +48,25 @@ Fortunately, there are a few ways to resolve this problem depending on which bra
 
 `master` is the main branch of your fork repository. It should contain a clean copy of `osu-wiki` repository's `master` branch contents without any personal edits. There are two ways to update the `master` branch of your fork:
 
+#### Using GitHub
+
+The wiki team has written a tool to allow you to synchronize your fork to the repository without ever leaving GitHub itself. The tool uses GitHub's continuous delivery feature called "GitHub Actions" and is self-contained within the repository.
+
+1. Go to the Actions tab of your fork.
+2. In "Workflows", look for "Sync from osu! upstream."
+3. Click "Run workflow". You will see the following options:
+
+![GitHub Actions Workflow - Run Workflow](img/github-actions-run-workflow.png "GitHub Actions Workflow - Run Workflow")
```

Comment:
These images are a bit small, I'd say.

Reply:
They're taken in a 1080p screen so it could use much more better screenshots in the near future.